sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn

```diff
@@ -1478,7 +1478,7 @@ or (calculatedDataPartnerFedralProvincialPDReceiptCost &gt; 0)</bpmn:conditionEx
     <bpmn:intermediateThrowEvent id="Event_1hl3pah" name="awardEligibilityCSGF">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=if (((calculatedDataFederalAssessedNeed + calculatedDataProvincialAssessedNeed) &#62;= 1) &#10;    and (programCredentialType = &#34;undergraduateCertificate&#34;&#10;         or programCredentialType = &#34;graduateCertificate&#34;&#10;         or programCredentialType = &#34;undergraduateDiploma&#34;&#10;         or programCredentialType = &#34;graduateDiploma&#34;&#10;         or programCredentialType = &#34;undergraduateDegree&#34;)&#10;    and (programLength = &#34;60WeeksToLessThan2Years&#34;&#10;         or programLength = &#34;2YearsToLessThan3Years&#34;&#10;         or programLength = &#34;3YearsToLessThan4Years&#34;&#10;         or programLength = &#34;4YearsToLessThan5Years&#34;&#10;         or programLength = &#34;5YearsOrMore&#34;)&#10;    and (calculatedDataTotalFamilyIncome &#60;= dmnFullTimeAwardFamilySizeVariables.limitAwardCSGFThresholdIncome)) &#10;then true else false" target="awardEligibilityCSGF" />
+          <zeebe:output source="=if (((calculatedDataFederalAssessedNeed + calculatedDataProvincialAssessedNeed) &#62;= 1) &#10;    and (programCredentialType = &#34;undergraduateCertificate&#34;&#10;		 or programCredentialType = &#34;undergraduateCitation&#34;&#10;         or programCredentialType = &#34;graduateCertificate&#34;&#10;         or programCredentialType = &#34;undergraduateDiploma&#34;&#10;         or programCredentialType = &#34;graduateDiploma&#34;&#10;         or programCredentialType = &#34;undergraduateDegree&#34;)&#10;    and (programLength = &#34;60WeeksToLessThan2Years&#34;&#10;         or programLength = &#34;2YearsToLessThan3Years&#34;&#10;         or programLength = &#34;3YearsToLessThan4Years&#34;&#10;         or programLength = &#34;4YearsToLessThan5Years&#34;&#10;         or programLength = &#34;5YearsOrMore&#34;)&#10;    and (calculatedDataTotalFamilyIncome &#60;= dmnFullTimeAwardFamilySizeVariables.limitAwardCSGFThresholdIncome)) &#10;then true else false" target="awardEligibilityCSGF" />
```

Comment:
The story says that in Camunda, that we should follow the same logic/path as `qualifyingStudies`.

![image](https://github.com/bcgov/SIMS/assets/54600590/571ae0db-2dbc-4236-8d0c-521f1e22d379)

But for  SABC Credentia it must follow the equivalent of `undergraduate certificate`. I would like to re-validate this point with business.

If we use the `undergraduateCitation` to follow path of qualifying studies in camunda. there is no PT impact.

![image](https://github.com/bcgov/SIMS/assets/54600590/f2b6da1c-bcd7-4523-a65c-416ae05751df)

Let me know if I am missing something.

Reply:
@dheepak-aot `undergraduate certificate` is used for "Field of Study" and `qualifyingStudies` for the eligibility.
The only concern from the AC perspective and that it mentions "Impacts are on PT and FT." as a complementary information.
Please confirm the business the intention but the PR is following the ACs, IMO.

Reply:
I'm confirming with @HRAGANBC

Reply:
I confirmed with Carly and did the changes to workflow. Thanks!